deisctl/units/deis-controller.service

```diff
@@ -1,14 +1,14 @@
 [Unit]
 Description=deis-controller
-Requires=fleet.socket
-After=fleet.socket
+Requires=docker.socket fleet.socket
+After=docker.socket fleet.socket
 
 [Service]
 EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-controller >/dev/null 2>&1 && docker rm -f deis-controller || true"
-ExecStart=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker run --name deis-controller --rm -p 8000:8000 -e EXTERNAL_PORT=8000 -e HOST=$COREOS_PRIVATE_IPV4 -v /var/run/fleet.sock:/var/run/fleet.sock $IMAGE"
+ExecStart=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker run --name deis-controller --rm -p 8000:8000 -e EXTERNAL_PORT=8000 -e HOST=$COREOS_PRIVATE_IPV4 -v /var/lib/docker:/var/lib/docker -v /var/run/docker.sock:/var/run/docker.sock -v /var/run/fleet.sock:/var/run/fleet.sock $IMAGE"
```

Comment:
The original version of this PR actually used a `docker` binary in deis-controller, so that's where the volume mount came from. Then I realized docker-py was sufficient, but didn't clean this up.